### DISCUSSION TITLE
Postal Mail delivery slow

### DISCUSSION THREAD

**Discussion by Shamri04** [postalserver/postal#2442] [Help with using Postal]
Hi, 

I am having a postal server with 16GB of RAM and 16 Cores CPU. And my worker quantity is 10 and the threads are 40.
But still, my mail delivery is very slow . Even I am not able to get 100 emails per minute.

![image](https://user-images.githubusercontent.com/110982013/235591313-3b9741e9-cb7b-401b-b0a6-da2cba0f9497.png)

**Comment by willpower232** [comment]
Unfortunately the config file values for worker quantity are ignored in v2 as Docker is controlling the processes. Have a look at #1604 for some direction on that.

**Reply by Shamri04** [reply]
Hi @willpower232 

Thanks for the info, tired starting the postal with a scale of 20 workers, will monitor the load for 1day and revert back

**Comment by Shamri04** [comment]
Hi @willpower232 

After scaling with 10 workers also not getting more than 14mails/per min.

![image](https://user-images.githubusercontent.com/110982013/236751159-26829bae-57ea-490f-bcf4-5972062e68f7.png)
